staging/src/k8s.io/cli-runtime/pkg/printers/omit_test.go

```diff
@@ -0,0 +1,106 @@
+/*
```

Comment:
@knight42 this one should match the rename, so `managedfields_test.go`

Reply:
Updated